packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -91,11 +95,11 @@ const BLOCK_PRODUCTION_RACE_CUTOFF_MS = 2_000;
 const BLOCK_PRODUCTION_RACE_TIMEOUT_MS = 12_000;
 
 type ProduceBlockOrContentsRes = {executionPayloadValue: Wei; consensusBlockValue: Wei} & (
-  | {data: allForks.BeaconBlock; version: ForkPreBlobs}
-  | {data: allForks.BlockContents; version: ForkBlobs}
+  | {data: BeaconBlock<ForkPreBlobs>; version: ForkPreBlobs}
+  | {data: BlockContents | SignedBlockContents; version: ForkBlobs}
```

Comment:
Why was `SignedBlockContents` added here? The beacon node can't produce signed blocks

```suggestion
  | {data: BlockContents; version: ForkBlobs}
```